Makefile

```
@@ -42,6 +42,10 @@ docs-clean:
 	uv run make -C docs clean
 	uv run make -C docs html
 
+.PHONY: docs-lite
+docs-clean:
+	uv run DOCS_LITE=1 make -C docs html
```

```suggestion
	uv run env DOCS_LITE=1 make -C docs html
```

This tells `uv` that `DOCS_LITE=1` is setting an environment variable.

---

Do you have a link to docs that show this? I'm not seeing anything here: https://docs.astral.sh/uv/reference/cli/#uv-run

---

I'm dumb, I didn't realize this is the standalone [`env`](https://www.man7.org/linux/man-pages/man1/env.1.html) command.

---

It is a [Linux command](https://www.man7.org/linux/man-pages/man1/env.1.html) and not part of `uv`. It appears that `uv` is expecting either a command, file, or directory and using `env` accomplishes this.